docs/guides/mbed_commissioning.md

```diff
@@ -0,0 +1,234 @@
+    -   [Sending Application packets for Android](#sending_app_data_android)
+    -   [Chip tool Linux](#tool_linux)
+    -   [Device Commisioning](#commisioni_linux)
+
+<hr> 
+
+
+<a name="overview"></a>
+## Overview
+
+
+This document provides step-by-step guide how to commission any CHIP application. 
+For demonstrating purpose the Lighting app is used. 
+
+Provisioning process is composed of the following  stages:
+
+ - CHIPTool discovers a CHIP accessory device over Bluetooth Low Energy (BLE).
+
+ - CHIPTool establishes a secure channel to the device over BLE, and sends network credentials  data.
+
+BLE is only used during first phase. Afterwards, only the IP connectivity between the smartphone and the accessory device is needed to send  messages.
+ 
+<a name="prerequisites"></a>
+### Prerequisites
+To complete all the steps in the tutorial, you need:
+
+-Smartphone with Android 8+ or PC with Ubuntu 20.04 and Bluetooth connectivity
+
+-any Wifi Access Point 
+-It can be smartfone router, standalone AP, wireless router or PC
+-Currently supported target device Cypress PSoC6 CY8CPROTO-062-4343W board
+
+<a name="tool_android"></a>
+### Chip tool for Android
+
+To make provisioning possible and control the CHIP  device from  Android based smartphone, CHIP Tool application must be built and installed first.
+ 
+To build the CHIPTool application for your smartphone, read
+[Android CHIPTool README](../../src/android/CHIPTool/README.md).
+
+After building, install the application by completing the following steps:
+
+1.  Install the Android Debug Bridge (adb) package by running the following
+    command:
+
+        $ sudo apt install android-tools-adb
+
+2.  Enable **USB debugging** on the smartphone. See the
+    [Configure on-device developer options](https://developer.android.com/studio/debug/dev-options)
+    guide on the Android Studio hub for detailed information.
+3.  If the **Install via USB** option is supported for your Android version,
+    turn it on.
+4.  Plug the smartphone into an USB port on your PC.
+5.  Run the following command to install the application, with _chip-dir_
+    replaced with the path to the CHIP source directory:
+
+        $ adb install -r chip-dir/src/android/CHIPTool/app/build/outputs/apk/debug/app-debug.apk
+
+6.  Navigate to settings on your smartphone and grant **Camera** and
+    **Location** permissions to CHIPTool.
+
+CHIPTool is now ready to be used for commissioning.
+
+After completing the app installation  main screen should appear. It's presented in the image below.
+
+
+ ![CHIPTool main screen](../images/CHIPTool_main.jpg)
+
+
+<a name="commisionig_setup_android"></a>
+##Device commisioning setup for Android 
+
+To prepare the accessory CHIP device for commissioning (called rendezvous), complete the following steps:
+
+ Use a terminal emulator to connect to the UART console of the accessory device.
+ For example Minicom or GNU Screen can be used.
+
+  minicom -D /dev/ttyACM0 -b 115200
```

Comment:
Place this in a code block (or code line)

Reply:
Done